localisation/00_wc_sad.csv

```diff
@@ -749,7 +749,21 @@ RULE_STATIC_PORTRAITS_ENABLED;Enabled;;Aktiviert;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_ENABLED_DESC;Static portraits for certain lore characters are �Genabled�!.;;Statische Portraits f�r bestimmte historische Charaktere sind �Gaktiviert�!.;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_DISABLED;Disabled;;Deaktiviert;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_DISABLED_DESC;Static portraits for certain lore characters are �Rdisabled�!.;;Statische Portraits f�r bestimmte historische Charaktere sind �Rdeaktiviert�!.;;;;;;;;;;;x
-;;;;;;;;;;;;;;
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
+writing_whispers;Hears Whispers;;;;;;;;;;;;;x
+writing_whispers_desc;This character hears whispers of the Old Gods.;;;;;;;;;;;;;x
+whispers_tooltip_tip;Quality of the book depends on amount of your �Yimpiety�!.;;;;;;;;;;;;;x
+EVTDESC_WCCSO.10200;The whispers of your masters have been getting stronger recently. They are pouring their eldritch knowledge into your mind but it's simply too much for your mind to take and your unable to make sense of any of it. You need to focus on portion of them and write them down but first you need a writing material.;;;;;;;;;;;;;x
+EVTOPTA_WCCSO.10200;That one servant girl had such a smooth skin.;;;;;;;;;;;;;x
+EVTDESC_WCCSO.10201;After spending some time trying to filter out one of the voices and gathering all needed materials you can now focus on one of the voices and write down its whispers.;;;;;;;;;;;;;x
+EVTOPTA_WCCSO.10201;It speaks of wars from long-forgotten eons.;;;;;;;;;;;;;x
+EVTOPTB_WCCSO.10201;It speaks of temple-cities made of black obsidian.;;;;;;;;;;;;;x
+EVTOPTC_WCCSO.10201;It speaks of creature with thousand forms.;;;;;;;;;;;;;x
+EVTOPTD_WCCSO.10201;It speaks of forgotten starry secrets.;;;;;;;;;;;;;x
+EVTDESC_WCCSO.10202_martial;As time progresses whispers get stronger and turn into visions and before long you can see numberless legions of the Void and terrible monstrosities designed to wage never-ending war in the name of their masters.;;;;;;;;;;;;;x
+EVTDESC_WCCSO.10202_stewardship;As time progresses whispers get stronger and turn into visions and before long you can see black temple-cities swarming with pyramids of incomprehensible geometry and obelisks portraying acts of eldritch depravity.;;;;;;;;;;;;;x
+EVTDESC_WCCSO.10202_intrigue;As time progresses whispers get stronger and turn into visions and before long you can see subterranean dungeons full of hooded beings scheming and plotting to advance their shadowy designs.;;;;;;;;;;;;;x
+EVTDESC_WCCSO.10202_learning;As time progresses whispers get stronger and turn into visions and before long you can see endless void filled with great spheres illuminated in twilight light with huge tendrils coming out of their cracked exteriors.;;;;;;;;;;;;;x
```

Comment:
As time progresses, the whispers get stronger and turn into visions and before long you can see the endless Void filled with great spheres illuminated in twilight with huge tendrils coming out of their cracked exteriors.